content/docs/studio/rest-api.md

```diff
@@ -103,6 +103,143 @@ Running this example will fail because it requires that you have [set up remote
+  name is specified, it will be extracted from repository name. This field is
+  optional.
+
+| header        | desc            | example value |
+| ------------- | --------------- | ------------- |
+| Authorization | Header for auth | token abc123  |
+
+Accepted formats for repository urls are:
+
+- `git@github.com:amritghimire/example-dvc-experiments.git`
+- `https://github.com/nimdraugsael/example-dvc-experiments.git`
+- `https://gitlab.example.org/amritghimire/example.git` (For custom gitlab)
+- `ssh://git@gitlab.com/xxx/yyy/zzz/repo.git`
+
+Similar format is supported for Github, Bitbucket and Gitlab.
+
+### Response
+
+Here is an example JSON response from the API request:
+
+```json
+{
+  "projects": [
+    {
+      "id": "tsanhawn9y",
+      "url": "https://studio.iterative.ai/user/amritghimire/projects/example-get-started-tsanhawn9y",
+      "name": "example-get-started"
+    }
+  ]
+}
+```
+
+This response confirms that the project was successfully created and provides
+the necessary information about the created project:
+
+- `id`: The unique identifier of the project. In this case, the project ID is
+  "tsanhawn9y".
+- `url`: The URL of the created project. You can access the project using this
+  URL. In this example, the project URL is
+  `https://studio.iterative.ai/user/amritghimire/projects/example-get-started-tsanhawn9y`.
+- `name`: The name of the created project. In this example, the project name is
+  "example-get-started".
+
+This information can be used to access and reference the project in the future.
+
+### Example
+
+First, you need your [DVC Studio access token]. For this example, we set it in
+the `DVC_STUDIO_TOKEN` environment variable:
+
+```sh
+export DVC_STUDIO_TOKEN=<TOKEN>
+```
+
+<toggle>
+
+<tab title="Python">
+
+```python
+import json
+import os
+import requests
+
+
+url = "https://studio.iterative.ai/api/create-project"
+token = os.environ["DVC_STUDIO_TOKEN"]
+payload = json.dumps({
+  "repo_url": "https://github.com/amritghimire/monorepo-model-registry-fixture",
+  "public": True,
+  "team_name": "team_name",
+  "subdir": "nested",
+  "name": "test-nested"
+})
+headers = {
+  'Authorization': f'token {token}',
+  'Content-Type': 'application/json'
+}
+
+response = requests.request("POST", url, headers=headers, data=payload)
+for project in response.json()["projects"]:
+    print(project["url"])
+    ...
+```
+
+</tab>
+
+<tab title="CLI">
+
+```sh
+$ curl --location 'https://studio.iterative.ai/api/create-project' \
+--header "Authorization:token ${DVC_STUDIO_TOKEN}" \
+--header 'Content-Type: application/json' \
+--data '{
+    "repo_url": "https://github.com/organization/repo",
+    "public": true,
+    "team_name": "team",
+    "subdir": "nested",
+    "name": "test-nested"
+}'
+```
+
+</tab>
+</toggle>
+
+<aside>
+💡 If you are interacting with Self hosted Studio, remember to replace the URL https://studio.iterative.ai with the URL of the self-hosted studio.
```

Comment:
style 

should be < 80

what is `aside`? we don't use it ... we have some other thing for notes - please check other docs

Reply:
Updated.